tst/performance/test_container_iterator.cpp

```diff
@@ -59,7 +59,8 @@ static const int N_kernels_to_launch_per_test = 100;
 template <typename InitFunc, typename PerfFunc>
 void performance_test_wrapper(const std::string test_name, InitFunc init_func,
                               PerfFunc perf_func) {
-  BENCHMARK_ADVANCED(test_name.c_str())(Catch::Benchmark::Chronometer meter) {
+  auto test_name_c = test_name.c_str();
+  BENCHMARK_ADVANCED(test_name_c)(Catch::Benchmark::Chronometer meter) {
```

Comment:
Oh, this change wasn't actually necessary - I was trying to figure out if this was confusing cpplint.